app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="space_8">8dp</dimen>
+
+    <dimen name="textsize_8">8dp</dimen>
+    <dimen name="textsize_12">12dp</dimen>
+    <dimen name="textsize_14">14dp</dimen>
```

Comment:
Text size should be in sp

Reply:
I use sp only in long paragraphs. In small cards with fixed size it can break all design (e.g. if user select large system font)

Reply:
If the size is fixed, the design will be broken. But I think you can write better. Don't forget about people with poor eyesight.

This is my design. Yes, now I see problems, but better for the visually impaired
<img src="https://user-images.githubusercontent.com/10943612/101981629-723c4e00-3c7f-11eb-8380-3e211af44831.png" width="15%"></img> <img src="https://user-images.githubusercontent.com/10943612/101981630-736d7b00-3c7f-11eb-890b-f4f409504da0.png" width="15%"></img>